inc/gui.class.php

```diff
@@ -33,7 +33,7 @@ public static function save_changes() {
 
 		$selected_languages_raw = wp_unslash( self::get_key( $_POST, 'ab_translate_lang' ) );
 		if ( ! is_array( $selected_languages_raw ) ) {
-			$selected_languages_raw = [];
+			$selected_languages_raw = array();
```

Comment:
Which php version is necessary? is the long string version of array necessary?
Currently we set it for PHP 5.2.4; however maybe we should switch. Have we more points to switch?

Reply:
The `array()` was introduced by merging the master again back into this branch. The reason was #209 but not coding standards itself. The coding standards do not restrict us to use `array()`. Although personally I am in favour of switching to a higher PHP version, we should be careful here, as we now know there are installations out there with PHP < 5.4 and we do not really know how many installations would be affected of such a step. But it is certainly worth to think about it.

Reply:
Yes, I think also we should carefully. Only if we have sufficient points to switch in a higher version we should switch. The codex should not a point. Ok, so we had to adjourn the topic of php version for the plugin.

Reply:
I don't think raising the minimum required PHP version is a step we should take in just one update. I have some thoughts on this, and might open a new issue for it :)